server/handlers/component_handler.go

```diff
@@ -79,49 +80,94 @@ func (h *Handler) GetAllComponents(rw http.ResponseWriter, r *http.Request) {
 	}
 }
 
-// swagger:route GET /api/meshmodel/components MeshmodelGet idMeshmodelGet
-// Handle GET request for getting all meshmodel components.
-// Components can be further filtered through query parameter ?version=
+/**Meshmodel endpoints **/
+const DefaultPageSizeForMeshModelComponents = 25
+
+// swagger:route GET /api/meshmodel/model GetMeshmodelModels idGetMeshmodelModels
+// Handle GET request for getting all meshmodel models. The component type/model name should be lowercase like "kubernetes", "istio"
+// ?version={version} If version is unspecified then all models are returned
+// ?order={field} orders on the passed field
```

Comment:
What's the difference between `order` and `sort`. I get what `sort` is, but it's not evident what `order` is here. Is this more like `filter` by type?

Reply:
Yes

Reply:
Got it. Please update these docs to better explain the difference. 

Please toss in an example or two showing usage; e.g. how POSTs can be used.

Reply:
model level POSTs are currently not supported as there are no registrants currently which register all entities combined at a model level

Reply:
POSTs are at Entity level currently and their docs are mentioned on the comments of their handlers.